themes/hello-friend-ng/layouts/wallets/list.html

```diff
@@ -1,74 +1,498 @@
+                        {
+                            label: 'Import Wallet',
+                            value: function(rowData, rowIdx) {
+                                if (rowData[5].match(/type_import_wallet.*/)) {
+                                    return rowData[5];
+                                }
+                            }
+                        },
+                        {
+                            label: 'Opensource',
+                            value: function(rowData, rowIdx) {
+                                if (rowData[5].match(/type_opensource.*/)) {
+                                    return rowData[5];
+                                }
+                            }
+                        },
+                        {
+                            label: 'MultiSig',
+                            value: function(rowData, rowIdx) {
+                                if (rowData[5].match(/type_multisig*/)) {
+                                    return rowData[5];
+                                }
+                            }
+                        },
+                        {
+                            label: 'API',
+                            value: function(rowData, rowIdx) {
+                                if (rowData[5].match(/type_api.*/)) {
+                                    return rowData[5];
+                                }
+                            }
+                        }
+                    ],
+                    combiner: 'and'
+                },
+                targets: [5],
+                                visible: false
+            }
+
+
+        ],
+        select: {
+            style:    'os',
+            selector: 'td:first-child'
+        }
+    });
+
+    dt.on('select.dt', () => {
+        dt.searchPanes.rebuildPane(0, true);
+    });
+
+    dt.on('deselect.dt', () => {
+        dt.searchPanes.rebuildPane(0, true);
+    });
+    table.searchPanes();
+    $("div.dtsp-verticalPanes").append(table.searchPanes.container());
+});
 </script>
 <main class="dogepedia">
-    <div class="title">
+    <div class="title" style="color: #FFFFFF">
       <h1>{{ .Title }}</h1>
       <a href="#" onclick="history.back()" style="text-decoration: none">< {{i18n "listWallets[muchgoback]" }} </a> — <b>{{ .Title }}</b></a>
     </div>
       <div class="underline"></div>
+
 <section class="box">
+    <main class="comic-neue">
 
   <!-- Tab content -->
-<div>
-<main class="dogepedia">
   <div style="max-width: 1200px; margin: auto;background-color: #FFFFFF !important; padding: 15px; border-radius: 20px">
-<table id="Wallets"cellspacing="1"  width="100%" style="border-color: #EEEEEE; text-align: left !important" class="table table-striped">
+            <div class="alert alert-warning" role="alert" style="width: 100%">
+          <i class="fa-solid fa-triangle-exclamation"></i> <b>Disclaimer:</b> We did not performed any audits to the current listed wallets below
```

Comment:
`We have not` or `We haven't` performed or `We did not perform` and instead of `any audits to the current listed wallets below`, change to `any audits of the wallets currently listed below`. Not sure we need `current` or `currently` either. Maybe `any audits of the wallets listed below` instead.

Reply:
Thanks @xanimo will review all your suggestions in a little bit after getting kids to sleep lol

Reply:
So I have applied many of your suggestions , don't know if the "Features colum will look great because it wastes to many space do to filters ":P 

![image](https://user-images.githubusercontent.com/5968076/182736935-eaf003cb-3991-4de7-aeef-cabecbfa19f5.png)

So in the filters I have fixed the colours and removed (forgot to do it) the search/order filters that is not needed for D0gecoin case, maybe in the future if we have more options to filter :)

![image](https://user-images.githubusercontent.com/5968076/182737107-91e046f5-a4ad-4d21-9954-9d7859f6e158.png)


Also added tooltips to help
![image](https://user-images.githubusercontent.com/5968076/182736115-c96b571b-8041-40d8-94f6-adc4366397af.png)

What do u think?

Reply:
This night I remember doing this :P
what do u think @xanimo ?
![image](https://user-images.githubusercontent.com/5968076/182803095-23aea70b-38af-4e2c-ae20-edcb02e4beeb.png)